_conferences/droids4droids-gt-2017.md

```diff
@@ -0,0 +1,10 @@
+---
+name: "Droids 4 Droids"
+website: http://gt.droids4droids.com/
+location: Guatemala, GT
```

Comment:
I don't quite understand this location.. shouldn't it be "Something, Guatemala"?